bindings/py/nupic/bindings/regions/TestNode.py

```diff
@@ -299,8 +302,11 @@ def getProtoType():
 
 
   def write(self, proto):
-    pass
-
+    regionImpl = proto.regionImpl.as_struct(TestNodeProto)
+    regionImpl.int32Param = self.getParameter('int32Param', 0)
```

Comment:
@chetan51 - I'm fine pushing the extra parameters off but can you follow up with that change right after we merge this? I don't want to forget about it.